pom.xml

```diff
@@ -29,6 +30,32 @@
             <name>Sonatype Public</name>
             <url>https://oss.sonatype.org/content/groups/public/</url>
         </repository>
+        <repository>
+            <id>ohdsi</id>
+            <name>repo.ohdsi.org</name>
+            <url>http://repo.ohdsi.org:8085/nexus/content/repositories/releases</url>
```

Comment:
This is where OHDSI have started publishing their releases as Maven artifacts. They don't have our version of the WebAPI published, which is why I published it [here](https://bintray.com/beta/#/phema/maven/ohdsi-webapi/2.6.0?tab=overview).